apps/modernization-ui/src/pages/advancedSearch/components/InvestigationResults.tsx

```diff
@@ -70,14 +70,24 @@ export const InvestigationResults = ({
         }
     };
 
+    const formattedName = (lastNm = '', firstNm = '') => {
```

Comment:
Can this function be pulled out into it's own file and imported when needed?  Then a unit test can be created to verify the output of the function